spec/deploy_dsl_spec.rb

```diff
@@ -53,7 +53,7 @@ class DeployDSLTest
     end
 
     it 'fails when an invalid capability is added' do
-      lambda{ DeployDSLTest.add_capability 'FOO_BAR' }.should raise_error SystemExit
+      expect { DeployDSLTest.add_capability 'FOO_BAR' }.to raise_error SystemExit
```

Comment:
I was getting a deprecation warning on this test.  This just brings it in  line with the expect syntax (test still passes).